File: problemas/iniciante/1154/1154.cs
using System;

class URI {
    static void Main(string[] args) {
        string entrada;
        int idade, soma, contador;

        soma = 0;
        contador = 0;
        while((entrada = Console.ReadLine()) != null){
            idade = int.Parse(entrada);

            if(idade < 0){
                break;
            }

            soma += idade;
            ++contador;
        }

        Console.WriteLine($"{(double)soma/contador:0.00}");
    }
}

File: problemas/iniciante/1154/1154.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = '';
let lineCount = 0;

process.stdin.on('data', (chunk) => {
  input += chunk;
});

process.stdin.on('end', () => {
  var lines = input.split('\n');

  let [soma, contador] = [0, 0];
  while (lines.length) {
    let idade = parseInt(lines.shift()!.trim());

    if (idade < 0) {
      break;
    }

    soma += idade;
    ++contador;
  }

  console.log(`${(soma / contador).toFixed(2)}`);
});